lib/ctx.cpp

```diff
@@ -451,9 +451,11 @@ grn_ctx_impl_init(grn_ctx *ctx)
                       ctx,
                       grn_msgpack_buffer_write);
 #endif
-  ctx->impl->output.arrow_stream_writer = NULL;
   grn_timeval_now(ctx, &ctx->impl->tv);
 #ifdef GRN_WITH_APACHE_ARROW
```

Comment:
We can remove this `#ifdef`.

Reply:
fix: [4e9ca04](https://github.com/groonga/groonga/pull/2399/commits/4e9ca04e8cc85ebd10c7f872f24c2aeefc755345)